core/src/com/unciv/logic/automation/Automation.kt

```diff
@@ -61,6 +61,9 @@ object Automation {
             for (unique in localUniqueCache.forCityGetMatchingUniques(city, UniqueType.UnhappinessFromPopulationTypePercentageChange))
                 if (unique.params[1] == "Specialists" && city.matchesFilter(unique.params[2]))
                     yieldStats.happiness -= (unique.params[0].toFloat() / 100f)  // relative val is negative, make positive
+            yieldStats.science *= 1.5f // we want to be working scientists
+            if (city.civ.getHappiness() < 0) // slotting Democracy specialists is an easy fix to happiness problems
+                yieldStats.happiness *= 2
```

Comment:
If you want to do something like this, look at line 57 for  the food-related unique. Otherwise you are just guessing that we have the democracy policy unlocked.

Reply:
Actually, we already have this implemented, line 61-63 accomplish this.